### PR TITLE
Calendar, new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: haiku-apps/calendar/calendar-1.0.recipe [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	makefile_engine
+	"
+
+BUILD()
+{
+	make $jobArgs
```
Comment (diversys) [inline haiku-apps/calendar/calendar-1.0.recipe]: make OBJ_DIR=objects $jobArgs

File: haiku-apps/calendar/calendar-1.0.recipe [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+INSTALL()
+{
+	mkdir -p "$appsDir"
+
+	if [ -n "$secondaryArchSuffix" ]; then
+		mv ./objects.x86-cc7-release/Calendar "$appsDir"
```
Comment (diversys) [inline haiku-apps/calendar/calendar-1.0.recipe]: Can be replaced with ```mv  objects/Calendar $appsDir```

File: haiku-apps/calendar/calendar-1.0.recipe [hunk 3]
```diff
@@ -0,0 +1,63 @@
+SUMMARY="A native Calendar application for Haiku"
+DESCRIPTION="A native Calendar application for Haiku
```
Comment (diversys) [inline haiku-apps/calendar/calendar-1.0.recipe]: No need to repeat summary.

File: haiku-apps/calendar/calendar-1.0.recipe [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+Event categorization.
+Set 'All day' long events.
+Fetching events from Google Calendar using Google Calendar API.
+SQLite backend for storing events.
+Setting preferences like 'First day of week', 'Display week number in \
+Calendar'.
+App localization: DateTime strings are localised and updates with locale \
+preferences change but GUI string still needs to be localized."
+HOMEPAGE="https://www.gnu.org/software/libcdio/"
+COPYRIGHT="2007-2017 Rocky Bernstein and Herbert Valerio Riedel"
```
Comment (scottmc) [inline haiku-apps/calendar/calendar-1.0.recipe]: Where did this info come from? The native Calendar app for Haiku was written during GSoC 2017.

File: haiku-apps/calendar/calendar-1.0.recipe [hunk 5]
```diff
@@ -0,0 +1,63 @@
+SUMMARY="A native Calendar application for Haiku"
+DESCRIPTION="A native Calendar application for Haiku
+Features:
+Create, modify and delete events.
```
Comment (diversys) [inline haiku-apps/calendar/calendar-1.0.recipe]: Add * before each line to have bullet points.

File: haiku-apps/calendar/calendar-1.0.recipe (hunk 4 above)
Comment (Begasus) [reply haiku-apps/calendar/calendar-1.0.recipe]: Jikes ;)

Comment (Begasus) [reply haiku-apps/calendar/calendar-1.0.recipe]: What would be the best to use here? COPYRIGHT="2017 GSoC"?

File: haiku-apps/calendar/calendar-1.0.recipe (hunk 3 above)
Comment (extrowerk) [reply haiku-apps/calendar/calendar-1.0.recipe]: DESCRIPTION=$SUMMARY , if you cannot find any better description...

File: haiku-apps/calendar/calendar-1.0.recipe (hunk 4 above)
Comment (scottmc) [reply haiku-apps/calendar/calendar-1.0.recipe]: 2017 Akshay Agarwal
2017-2018 Haiku Archives Team

**Pull request merged**